template/greeter/Dockerfile

```diff
@@ -7,5 +7,5 @@ WORKDIR /workdir
 COPY package.json yarn.lock ./
```

Comment:
We would include `.npmrc` in this `COPY` once we switch the templates over to pnpm